packages/live-preview-sdk/src/index.ts

```diff
@@ -374,7 +374,7 @@ export class ContentfulLivePreview {
       return;
     }
 
-    debug.error('Please provide field id and entry id to openEntryInEditor.');
+    debug.error('Please provide field id and entry/asset id to openEntryInEditor.');
```

Comment:
Can we provide an asset id to `openEntryInEditor` ? 🤔 Shouldn't it be renamed then to `openEntityInEditor`

Reply:
We can indeed, and that would make more sense, although I wonder if it would be a breaking change 🤔

Reply:
Added a todo since it would be a breaking change, and also added the props to this error